internal/test/builders/service.go

```diff
@@ -6,20 +6,12 @@ import (
 
 // Service creates a service with default values.
 // Any number of service builder functions can be passed to augment it.
-// Currently, only ServiceName is implemented
 func Service(builders ...func(*swarm.Service)) *swarm.Service {
-	service := &swarm.Service{
-		ID: "serviceID",
-		Spec: swarm.ServiceSpec{
-			Annotations: swarm.Annotations{
-				Name: "defaultServiceName",
-			},
-			EndpointSpec: &swarm.EndpointSpec{},
-		},
-	}
+	service := &swarm.Service{}
+	defaults := []func(*swarm.Service){ServiceID("serviceID"), ServiceName("defaultServiceName")}
```

Comment:
Actually thinking that we could even remove these defaults; looks like it's currently not used in other locations where the code depends on it being set (which means we could convert this package to something that can be used _outside_ of testing as well